jobs/config.json

```diff
@@ -4448,16 +4448,18 @@
   },
   "ci-kubernetes-e2e-gci-gke": {
     "args": [
+      "--check-leaked-resources",
+      "--cluster=",
```

Comment:
Honestly, having a unique cluster name is helpful when debugging on the backend, too (o/w it ends up as `bootstrap-e2e`, which is not super helpful). We can, of course, find the logs for that cluster through other means (i.e. timestamp and what project it was running in), but name is fast.